.github/workflows/smoketest.yml

```diff
@@ -290,6 +290,12 @@ jobs:
         `../src/vcpkg/vcpkg fetch cmake | tail -n 1` ../src -Wno-dev -DRUNTIME_USER=$(whoami) -DRUNTIME_GROUP=$(id -gn) -DDESTDIR=$(realpath ..)/install -DINCLUDE_PLUGINS=1 -DCMAKE_BUILD_TYPE=Release -DUSE_PYTHON2=0 -DUSE_PYTHON3=1 -DSUPPRESS_SPARK=1 -DUSE_CPPUNIT=1 -DUSE_CASSANDRA=Off -DCMAKE_CXX_FLAGS="-DBUILD_TAG=\\\"dummytag\\\" -DBUILD_VERSION_MAJOR=1 -DBUILD_VERSION_MINOR=1 -DBUILD_VERSION_POINT=0" -DBUILD_TAG="smoketest" -DSTRIP_RELEASE_SYMBOLS=0 -DCMAKE_CXX_FLAGS="-DTRACE_GLOBAL_GROUP=1"
         make -j$(nproc) install
 
+    - name: Run error handling script
+      run: |
+        chmod +x ./.github/workflows/johnnytest4.sh
```

Comment:
In GitHub runner the directory structure is a bit different than as is in BM/VM. So, to use your script it is better to copy into the current directory:
```
cp -v ${{ github.workspace }}/src/.github/workflows/johnnytest4.sh .
```
then do the other things.